infrastructures/infrastructure-aws-ec2/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AWSEC2Infrastructure.java

```diff
@@ -122,36 +122,36 @@
     @Configurable(description = "The number of nodes to create on each VM", sectionSelector = 2, important = true)
     protected int numberOfNodesPerInstance = 1;
 
-    @Configurable(description = "(optional, default value: " + DEFAULT_IMAGE +
-                                ") VM image id, format region/imageId", sectionSelector = 3)
+    @Configurable(description = "VM image id, format region/imageId (optional, default value: " + DEFAULT_IMAGE +
+                                ")", sectionSelector = 3, important = true)
     protected String image = DEFAULT_IMAGE;
 
-    @Configurable(description = "(optional, default value: " + DEFAULT_VM_USERNAME +
-                                ") Default username of your VM image, make sure it's adapted to 'image'", sectionSelector = 3)
+    @Configurable(description = "Default username of your VM image, make sure it's adapted to 'image' (optional, default value: " +
+                                DEFAULT_VM_USERNAME + ")", sectionSelector = 3)
```

Comment:
It the string ending with `+` on purpose ? Same comment for next descriptions.

Reply:
It's not end with `+`, there is a `)` at the end.